int64 object_index
---
float64[] object_state
